pkg/macros/macros_test.go

```diff
@@ -93,3 +114,31 @@ func TestMacroIntervalSeconds(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "20", got)
 }
+
+// test sqlds query interpolation with clickhouse filters used
+func TestInterpolate(t *testing.T) {
```

Comment:
I copied the test we have in sqlds but here it's running with clickhouse specific macros